endpoint.go

```diff
@@ -637,6 +639,14 @@ func EndpointOptionGeneric(generic map[string]interface{}) EndpointOption {
 	}
 }
 
+// CreateOptionIpam function returns an option setter for the ipam configuration for this endpoint
+func CreateOptionIpam(prefAddress net.IP, ipamOptions map[string]string) EndpointOption {
```

Comment:
I don't think that should be the job of libnetwork though. I understand the need for a VIP and a local address but the "docker ecosystem" gets rid of the need for a VIP by doing some sort of service discovery.